WORKSPACE

```diff
@@ -1230,3 +1230,47 @@ go_repository(
     remote = "https://github.com/cf-routing/certstrap",
     vcs = "git",
 )
+
+go_repository(
+    name = "io_k8s_helm",
+    commit = "8478fb4fc723885b155c924d1c8c410b7a9444e6",
+    importpath = "k8s.io/helm",
+)
+
+go_repository(
+    name = "com_github_Masterminds_sprig",
+    commit = "b217b9c388de2cacde4354c536e520c52c055563",
+    importpath = "github.com/Masterminds/sprig",
+)
+
+go_repository(
+    name = "com_github_Masterminds_semver",
+    commit = "517734cc7d6470c0d07130e40fd40bdeb9bcd3fd",
+    importpath = "github.com/Masterminds/semver",
+)
+
+go_repository(
+    name = "com_github_aokoli_goutils",
+    commit = "3391d3790d23d03408670993e957e8f408993c34",
+    importpath = "github.com/aokoli/goutils",
+    remote = "https://github.com/Masterminds/goutils",
+    vcs = "git",
+)
+
+go_repository(
+    name = "com_github_gobwas_glob",
+    commit = "bea32b9cd2d6f55753d94a28e959b13f0244797a",
+    importpath = "github.com/gobwas/glob",
+)
+
+go_repository(
```

Comment:
Do we really need to add so many new dependencies?

Reply:
Yeah, I was pretty bummed - they're all due to transitive dependencies of the helm rendering libraries.

Reply:
Please make sure 'dep ensure' works, and if any pinning is needed - document it.